Userland/Libraries/LibJS/Bytecode/ASTCodegen.cpp

```diff
@@ -1004,32 +1004,24 @@ void IfStatement::generate_bytecode(Bytecode::Generator& generator) const
     Bytecode::Op::Jump* true_block_jump { nullptr };
 
     generator.switch_to_basic_block(true_block);
+    generator.emit<Bytecode::Op::LoadImmediate>(js_undefined());
     m_consequent->generate_bytecode(generator);
     if (!generator.is_current_block_terminated())
         true_block_jump = &generator.emit<Bytecode::Op::Jump>();
 
     generator.switch_to_basic_block(false_block);
-    if (m_alternate) {
-        auto& end_block = generator.make_block();
+    auto& end_block = generator.make_block();
```

Comment:
This makes the comment above about not generating the end block incorrect.
Please also fixup the comment (by removing it, since it no longer does that)

Reply:
Fixed. :)